wai-logger/Network/Wai/Logger.hs

```diff
@@ -96,26 +96,17 @@ data ApacheLoggerActions = ApacheLoggerActions {
   , logRemover :: IO ()
   }
 
--- | Logger Type.
-data LogType = LogNone                     -- ^ No logging.
-             | LogStdout BufSize           -- ^ Logging to stdout.
-                                           --   'BufSize' is a buffer size
-                                           --   for each capability.
-             | LogFile FileLogSpec BufSize -- ^ Logging to a file.
-                                           --   'BufSize' is a buffer size
-                                           --   for each capability.
-                                           --   File rotation is done on-demand.
-             | LogCallback (LogStr -> IO ()) (IO ())
-
 ----------------------------------------------------------------
 
```

Comment:
Turns out yesod-core is using DateCacheGetter: https://github.com/yesodweb/yesod/issues/1214